test/run-openshift

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Test the NodeJS image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+
+source ${THISDIR}/test-lib.sh
+source ${THISDIR}/test-lib-openshift.sh
```

Comment:
Double quote this please.

Reply:
Also `${IMAGE_NAME}` needs double quoting in the `ct_os_test_s2i_app` invocations

Reply:
Thanks, quotes added.